config.js

```diff
@@ -1,4 +1,4 @@
-export const testSubjects = ['rollup', 'webpack', 'parcel', 'browserify'];
+export const testSubjects = ['browserify', 'parcel', 'rollup', 'webpack'];
```

Comment:
One part of the code was calling `testSubjects.sort()`, forgetting it was a mutable operation. Instead, I've changed the order here to be the 'desired order'.